src/components/CheckIn/CheckIn.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import Route from 'react-router-dom/Route';
+
+import {
+  Button,
+  ButtonGroup,
+  Col,
+  IconButton,
+  Pane,
+  PaneMenu,
+  Paneset,
+  Row,
+} from '@folio/stripes/components';
+import { stripesShape } from '@folio/stripes/core';
+
+import { ORDER } from '../Utils/resources';
+import CheckInItems from './CheckInItems';
+import CheckInHistory from './CheckInHistory';
+
+export const ITEMS = '/items';
+export const HISTORY = '/history';
+
+class CheckIn extends Component {
```

Comment:
why not to use `PureComponent`?

Reply:
Because it's performance optimization, and premature optimization is a root of all evils, as you may know) I don't think that's it has slow renderings right now, so no need to make react compare things and waste time more than just to render them. What do you think?

Reply:
optimization from react, but I don't think now it's suitable for this component as there are conditions for `PureComponent`